.github/actions/install-gitea/gitea-config.yaml

```diff
@@ -6,10 +6,3 @@ redis:
   enabled: true
 postgresql:
   enabled: true
-service:
-  http:
-    type: NodePort
-    nodePort: 30080
```

Comment:
we can remove these now because we're accessing the gitea server via port-forward instead of nodeport